docker-compose.yml

```diff
@@ -119,4 +119,4 @@ services:
       - COOKIES_KEY=topsecret
       - JWT_SECRET='|r+.2!!!.Qf_-|63*%.D'
       - JWT_AUDIENCE='Daily Staging'
-      - JWT_ISSUER='Daily API Staging'
```

Comment:
Why was there a change here and in the `prettier.xml`?

Reply:
This is one updated automatically (I've been running prettier for all files) and it just inserted a new line. Regarding prettier.xml - you can see this one inside` .idea` folder and it's used by JetBrains IDEs to keep configurations. Usually it's added to .gitgnore, but not in our case and by default prettier is not working on save action what I'm usually change - because of this prettier.xml file was updated and added to git changelist. We should probably add .idea folder to .gitignore, what do you think @rebelchris?

Reply:
Ah perfect lets exclude the idea folder for now yes :)